project/Paradox.scala

```diff
@@ -23,26 +23,27 @@ import sbt._
 import scala.concurrent.duration._
 
 object Paradox {
-
+  val pekkoBaseURL = "https://pekko.apache.org"
   val propertiesSettings = Seq(
     Compile / paradoxProperties ++= Map(
-      "canonical.base_url" -> "https://doc.akka.io/docs/akka/current",
+      "canonical.base_url" -> s"$pekkoBaseURL/docs/pekko/current",
       "github.base_url" -> GitHub
         .url(version.value), // for links like this: @github[#1](#1) or @github[83986f9](83986f9)
-      "extref.pekko.http.base_url" -> "https://doc.akka.io/docs/akka-http/current/%s",
-      "extref.pekko-management.base_url" -> "https://doc.akka.io/docs/akka-management/current/%s",
-      "extref.platform-guide.base_url" -> "https://developer.lightbend.com/docs/akka-platform-guide/%s",
+      "extref.pekko.http.base_url" -> s"$pekkoBaseURL/docs/pekko-http/current/%s",
+      "extref.pekko-management.base_url" -> s"$pekkoBaseURL/docs/pekko-management/current/%s",
+      "extref.platform-guide.base_url" -> s"$pekkoBaseURL/",
       "extref.wikipedia.base_url" -> "https://en.wikipedia.org/wiki/%s",
       "extref.github.base_url" -> (GitHub.url(version.value) + "/%s"), // for links to our sources
       "extref.samples.base_url" -> "https://developer.lightbend.com/start/?group=akka&amp;project=%s",
```

Comment:
I don't think `extref.samples.base_url` exist. This results `404`s on lightbend side because `.*pekko.*` project does not exists on their side. For example, 

```
https://developer.lightbend.com/start/?group=akka&project=pekko-sample-cluster-docker-compose-scala
```

Reply:
just add issues for anything you are skipping

Reply:
Added an issue https://github.com/apache/incubator-pekko/issues/124